packages/mrm-core/src/npm.js

```diff
@@ -90,6 +91,25 @@ function uninstall(deps, options = {}, exec) {
 	return run(newDeps, { remove: true, dev }, exec);
 }
 
+/**
+ * Return suitable run function
+ *
+ * @param {Options} [options]
+ */
+function defineRun(options = {}) {
```

Comment:
Let's avoid reassignments in this function (there's no reason for them), and give it a better name, maybe `getRunFunction`, since it's returning a function, not defining it.

Reply:
Sure, fixed